inst/include/Rcpp/Reference.h

```diff
@@ -59,15 +59,15 @@ namespace Rcpp{
         Reference_Impl( const std::string& klass ) {
             SEXP newSym = Rf_install("new");
             Shield<SEXP> call( Rf_lang2( newSym, Rf_mkString( klass.c_str() ) ) );
-            Storage::set__( Rcpp_eval( call ) ) ;
+            Storage::set__( Rcpp_eval( call , Rcpp::internal::get_Rcpp_namespace()) );
```

Comment:
I'm guessing we avoid depending on `methods` because we only need it when modules are used, and not all Rcpp users require modules?

Either way, I think this is fine.